crates/codegen/language/definition/src/model/manifest.rs

```diff
@@ -144,3 +147,35 @@ pub enum BuiltInLabel {
     LeadingTrivia,
     TrailingTrivia,
 }
+
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
+#[derive_spanned_type(Clone, Debug, EnumDiscriminants, ParseInputTokens, WriteOutputTokens)]
+pub enum BuiltIn {
```

Comment:
IIRC, you suggested that built-in functions would not version their parameters for simplicity in codegen, and we would just define another function if a parameter changes between versions.

Is that the same case for type fields? are they allowed to be versioned under the same type? or would we be defininig multiple types?